src/lib.rs

```diff
@@ -320,6 +327,36 @@ pub async fn compare_attestations(
     let mut remote_attestations = remote_attestations.clone();
     remote_attestations.sort_by(|a, b| a.stake_weight.partial_cmp(&b.stake_weight).unwrap());
 
+    if !remote_attestations.is_empty() {
+        let subgraph_gauge = ACTIVE_SUBGRAPHS.with_label_values(&[&ipfs_hash]);
+
+        let all_senders: Vec<String> = remote_attestations
```

Comment:
What do you think about tracking both total and unique number of sender? There shouldn't be duplicated senders for a block+ipfs, but if so this could help expose the error?

Reply:
great idea! Can we leave it for the follow-up PR though?

Reply:
okay! will paste a link to this comment in the prometheus improvement issue